src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -9,11 +9,11 @@ internal class LocalizableStrings
 
         public const string AppDescription = "Command to add package reference";
 
-        public const string CmdPackageDescription = "Package references to add";
+        public const string CmdPackageDescription = "Package reference to add";
```

Comment:
we need to start adding the missing punctuation too. Also wouldn't this read better as "The package reference to add."?